src/course-unit/CourseUnit.test.jsx

```diff
@@ -218,6 +218,53 @@ describe('<CourseUnit />', () => {
       .toHaveBeenCalledWith(`/course/${courseId}/container/${blockId}/${updatedAncestorsChild.id}`, { replace: true });
   });
 
+  it('after changing the unit header, the sequence unit is updated', async () => {
```

Comment:
`'the sequence unit is updated after changing the unit header'`

Reply:
Renamed